package/kedro_viz/api/graphql.py

```diff
@@ -127,6 +207,12 @@ class TrackingDataSet:
 class Query:
     """Query endpoint to get data from the session store"""
 
+    @strawberry.field
+    def run_tracking_data(self, run_ids: List[ID]) -> List[TrackingDataSet]:
+        """Query to get data for specific runs from the session store"""
+        runs = get_run_tracking_data(run_ids)
+        return runs
```

Comment:
```suggestion
        return get_run_tracking_data(run_ids)
```

Why do we need to have a method called `get_run_tracking_data`? Can't the logic be written inside this resolver instead? Is it to make it easier to test?